src/markup/patterns/drizzle/partials/sidebar.hbs

```diff
@@ -3,49 +3,33 @@ hidden: true
 ---
 {{#*inline "nav-item"}}
   <a class="drizzle-nav__link drizzle-Nav-item" href="{{@root.baseurl}}/{{url}}">{{label}}</a>
-{{/inline}}
+  {{/inline}}
 
-<div class="drizzle-Layout-nav drizzle-nav" id="nav">
-  <div class="drizzle-nav__header">
-    <p class="drizzle-nav__title"><a href="/" class="x">{{data "global.companyName"}} Design&nbsp;System</a></p>
+  <div class="drizzle-Layout-nav drizzle-nav" id="nav">
+    <div class="drizzle-nav__header">
+      <p class="drizzle-nav__title"><a href="/" class="x">{{data "global.companyName"}} Design&nbsp;System</a></p>
 
-    <button class="drizzle-nav__toggle c-expandable-content__button js-toggler" id="nav-content">Menu</button>
+      <button class="drizzle-nav__toggle c-expandable-content__button js-toggler" id="nav-content">Menu</button>
 
-    <!-- <a href="/" class="drizzle-Nav-title">{{data "global.companyName"}} Design System</a>
+      <!-- <a href="/" class="drizzle-Nav-title">{{data "global.companyName"}} Design System</a>
 
```

Comment:
I feel as though the indentation on this file was good in the original state but I could be wrong! Thoughts, @zastrow ?

Reply:
Fixed!